### ISSUE TITLE
Single code base FAQ is problematic

### ISSUE THREAD

**Issue by alakoring** [Microsoft/WinObjC#2007]
I’m trying to get my iOS app to build, having just one copy of source. (Using VMware on my Mac, and the 1612 VM.)

The answer at

https://github.com/Microsoft/WinObjC/wiki/FAQ#how-can-i-use-a-vm-on-a-mac-to-develop-a-single-codebase-simultaneously-in-visual-studio-and-xcode

generates

Warning	MSB8029	The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build.	CocoaLumberjack-iOS	C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppBuild.targets	400

**Comment by antoine-haas** [comment]
Hello @alakoring,

Are you facing this issue only when trying to build on a shared folder between your Mac and the VM, or also when trying to build on a non-shared folder on the VM?

**Comment by alakoring** [comment]
Um, the FAQ was about building with a shared folder…

I converted them all to non-$(Temp) paths and got rid of the warning. But then I happened to copy everything out of the shared folder and get

Warning	MSB8029	The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build.	Six Ages	C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppBuild.targets	400	

Don't know if I missed that because of #2008 or what, but the message seems to say: never use $(Temp).

**Comment by antoine-haas** [comment]
Using $(Temp) does indeed trigger the warning above, but this shouldn't prevent you from building the app. There must be another error in your build error list. As you pointed out, changing the Output directory path so that it's not under $(Temp) should get rid of the warning.

**Comment by alakoring** [comment]
Right, the bug is against the documentation. (Which wasn’t super clear, I think I was trying to say that I was just starting and having a hard time because the directions weren’t correct.)
